pkg/skaffold/tag/git_commit.go

```diff
@@ -135,6 +136,15 @@ func gitAbbrevtreesha(ctx context.Context, workingDir string) (string, error) {
 	return runGit(ctx, workingDir, "rev-parse", "--short", "HEAD:"+gitPath+"/")
 }
 
+func gitBranches(ctx context.Context, workingDir string) (string, error) {
+	gitBranch, err := runGit(ctx, workingDir, "branch", "--show-current")
+	if err != nil {
+		return gitAbbrevcommitsha(ctx, workingDir)
```

Comment:
can you add a warning message here that `branches` tagger failed and we're falling back to `abbrevcommitsha`

Reply:
Is this a warning condition? e.g. afaik the `Tags` variant doesn't consider it a warning when it falls back to the abbrevcommitsha, the only difference is that they found a `git` command that automatically falls back (that's the --always), whereas I couldn't find a single git shell command, so I created one by explicitly falling back

Reply:
looks good as is 👍